cli/command/inspect/inspector.go

```diff
@@ -84,6 +87,37 @@ func Inspect(out io.Writer, references []string, tmplStr string, getRef GetRefFu
 	return nil
 }
 
+// All fetches all objects using GetAllFunc and writes the json
+// representation to the output writer.
+func All(out io.Writer, tmplStr string, getAll GetAllFunc) error {
+	inspector, err := NewTemplateInspectorFromString(out, tmplStr)
+	if err != nil {
+		return cli.StatusError{StatusCode: 64, Status: err.Error()}
```

Comment:
What is the significance of 64?

Reply:
It was just to keep consistent with the Inspect function. Code duplication removed.